src/globals/scss/_feature-flags.scss

```diff
@@ -4,7 +4,7 @@
 
 // Initialize our feature flag map with default values
 $feature-flags: (
-  ui-shell: false,
+  ui-shell: true,
```

Comment:
make sure this gets switched back to false before merging

Reply:
yes! thanks for catching that

Reply:
Only because @joshblack caught it in my PR 😉